src/main/java/org/gridsuite/filter/server/utils/expertfilter/OperatorType.java

```diff
@@ -10,6 +10,7 @@
  * @author Antoine Bouhours <antoine.bouhours at rte-france.com>
  */
 public enum OperatorType {
+    UNKNOWN, // used for test covering
```

Comment:
I didn't notice we already have an UNKNOW for FieldType, but for me it's a bad practice to add code for test purpose.
Instead, you can use a combination which is not implemented.

Reply:
DONE! All UNKNOWN(s) have been removed, use adequate combination to test exceptions